### PR TITLE
Fixing last mentions of Http and note for angular <4.3 users

### PR DESCRIPTION
**Pull request by Murazaki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,12 @@ Now you need to install the npm module for `TranslateHttpLoader`:
 npm install @ngx-translate/http-loader --save
 ```
 
+**NB: if you're still on Angular >4.3, please use Http from @angular/http with http-loader@0.1.0.**
```
Comment (SamVerschueren) [inline README.md]: `Angular <4.3`

**Review by SamVerschueren — CHANGES_REQUESTED** [review]
One small change

**Review by SamVerschueren — APPROVED** [review]

**Comment by ocombe** [comment]
Thanks, I'm on holidays and my connection here is horrible, I cannot even check out the repo...

**Comment by Murazaki** [comment]
Fix #20

**Pull request merged**